fhir-validation/pom.xml

```diff
@@ -40,6 +40,12 @@
             <artifactId>fhir-examples</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>fhir-ig-us-core</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
circular dependency, but i think its ok because its just test scope.  but still might be better to have us-core-specific tests under the us-core project.

Reply:
I think this should be okay given the test scope. I'm using us-core IG to validate the validator / constraint generator. Once that work has been wrapped up for 4.0.0, I may consider moving it out into a different project.